docs/hello_nextflow/03_hello_gatk.md

```diff
@@ -228,40 +222,54 @@ process GATK_HAPLOTYPECALLER {
 }
 ```
 
-#### 2.2. Add accessory inputs up top
+#### 2.2. Add definitions for accessory inputs
 
 ```groovy title="hello-gatk.nf"
 // Accessory files
-params.genome_reference = "${projectDir}/data/ref/ref.fasta"
-params.genome_reference_index = "${projectDir}/data/ref/ref.fasta.fai"
-params.genome_reference_dict = "${projectDir}/data/ref/ref.dict"
-params.calling_intervals = "${projectDir}/data/intervals.list"
+params.reference        = "${projectDir}/data/ref/ref.fasta"
+params.reference_index  = "${projectDir}/data/ref/ref.fasta.fai"
+params.reference_dict   = "${projectDir}/data/ref/ref.dict"
+params.intervals        = "${projectDir}/data/ref/intervals.bed"
+```
+
+#### 2.3. Make a value channel for each of the accessory files
+
+Add this to the workflow block (after the `reads_ch` creation):
+
+```groovy title="hello-gatk.nf"
+// Create channels for the accessory files (reference and intervals)
+ref_file        = file(params.reference)
+ref_index_file  = file(params.reference_index)
+ref_dict_file   = file(params.reference_dict)
+intervals_file  = file(params.intervals)
```

Comment:
```suggestion
ref_file        = Channel.value(file(params.reference))
ref_index_file  = Channel.value(file(params.reference_index))
ref_dict_file   = Channel.value(file(params.reference_dict))
intervals_file  = Channel.value(file(params.intervals))
```
Shouldn't we do this to properly create Channels

Reply:
@adamrtalbot suggested this syntax

Reply:
> I'd like to elaborate on value vs queue channels in a full module dedicated to channels (hello-channels).

Love this idea-- definitely want to keep it simple here, especially since those inputs are boring

Reply:
Then, let's not comment that we're creating Channels, we're not